public/locales/en_US/optimizerTab.yaml

```diff
@@ -319,16 +319,18 @@ SetConditionals:
     Prisoner: The selected DEF% pen buff is applied to damage calculations based on the number of stacks.
     Diver: The 2 piece effect can be disabled by selecting the 0x option. For 4 piece, different CRIT buffs are applied to Combat stat calculations depending on the selected option.
     Watchmaker: When enabled, the Break Effect buff is applied to Combat stat calculations.
-    Valorous: The CRIT Rate buff is always on by default. The selected buff is applied to damage calculations based on the number of stacks.
+    Valorous: The CRIT Rate buff is always on by default. When enabled, the DMG% buff for Ultimate damage is applied to Combat stat calculations.
+    Sacerdos: The selected buff is applied to damage calculations. Characters who buff themselves can trigger this effect.
+    Scholar: The CRIT Rate buff is always on by default. When enabled, the DMG% buff for Skill damage will be enhanced in accordance with the character's COMBO sequence.
+    Hero: The ATK% buff is always on by default. When enabled, the SPD% and CRIT Damage buffs are applied to combat stat calculations.
```

Comment:
These are technically only supposed to be for the 4p, for example Scholar description doesnt mention cr at all

```
Scholar Lost in Erudition
Set description

Increases DMG dealt by Skill and Ultimate by 20%. After using Ultimate, additionally increases the DMG dealt by the next Skill by 25%.
```